pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -48,7 +48,9 @@ in
 # We cannot enable -j<n> parallelism for libraries because GHC is far more
 # likely to generate a non-determistic library ID in that case. Further
 # details are at <https://github.com/peti/ghc-library-id-bug>.
-, enableParallelBuilding ? (stdenv.lib.versionOlder "7.8" ghc.version && !isLibrary) || stdenv.lib.versionOlder "8.0.1" ghc.version
+#
```

Comment:
Are you using this to force a section break or should this line be removed?

Reply:
It's just a leftover from a rebase. If the owners agree with the substantive changes, I'll fix it.